### PR TITLE
Add Members + Avatar command

### PR DESCRIPTION
**Pull request by milanmdev**

### CODE REVIEW

File: commands/members.js [hunk 1]
```diff
@@ -0,0 +1,7 @@
+module.exports.run = async function () {
+    return this.reply(`**Member Count:** ${message.guild.members.cache.filter((m) => !m.bot).size.toLocaleString()}`);
+}
+
+module.exports.info = {
+    aliases: ['members']
```
Comment (jpbberry) [inline commands/members.js]:
```suggestion
```
Aliases don't need to be added when the file name is already the command
(Do the same for avatar command)

**Review by jpbberry — CHANGES_REQUESTED** [review]

**Comment by milanmdev** [comment]
Done!

**Comment by milanmdev** [comment]
Fixed again 👀

**Review by jpbberry — APPROVED** [review]

**Comment by jpbberry** [comment]
I'll squash merge this once we've fixed the issue with the bot not restarting.

Thanks for contrib!

**Comment by milanmdev** [comment]
aight. and np!

**Pull request merged**